quiz.js

```diff
@@ -17,9 +17,22 @@ db.dropDatabase();
 
 // your code begins here
 
+use houses
 
+db.houses.insert({
+  name: 'As High as Honor',
+  motto: 'female'
+});
```

Comment:
Whoops - I think you may have slipped up on this one. Should have been 
`{
  name: 'Arryn',
  motto: 'As High as Honor'
}`
But your syntax is all correct.